bitnami/logstash/README.md

```diff
@@ -67,6 +67,8 @@ The following tables lists the configurable parameters of the Logstash chart and
 | `enableMonitoringAPI`                      | Whether to enable the Logstash Monitoring API or not  Kubernetes cluster domain                                      | `true`                                                  |
 | `monitoringAPIPort`                        | Logstash Monitoring API Port                                                                                         | `9600`                                                  |
 | `extraEnvVars`                             | Array containing extra env vars to configure Logstash                                                                | `nil`                                                   |
+| `envFrom.secretRef.name`                   | Kubernetes Secrets name                                                                                             | `nil`                                                   |
+| `envFrom.configMapRef.name`                | Kubernetes Configmap name                                                                                           | `nil`                                                   |
```

Comment:
Thank you so much for the PR! In our other charts, what we use is the value extraEnvVarsSecret and extraEnvVarsCM. See an example here: https://github.com/bitnami/charts/blob/master/bitnami/dokuwiki/templates/deployment.yaml#L104

Could you adapt it so it is consistent with the rest of the charts?

Reply:
@javsalgar That makes sense. I've made the change.